qutebrowser/utils/usertypes.py

```diff
@@ -297,11 +297,16 @@ def __init__(self, fileobj):
 
 class OpenFileDownloadTarget(DownloadTarget):
 
-    """Save the download in a temp dir and directly open it."""
+    """Save the download in a temp dir and directly open it.
 
-    def __init__(self):
+    Attributes:
+        cmdline: The command to use as string. A {} is expanded to the
```

Comment:
I've not added backticks here because this is no command, the backticks were mainly for asciidoc. Should I still add them?